.github/workflows/compare_config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       diff_detected: ${{steps.config_diff.outputs.diff}}
       artifact_url: ${{steps.save_diff.outputs.artifact-url}}
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
good one :)

Reply:
Github made a warning that it will change the image under the latest variant to 24.04, so just to avoid wall of warnings, I manually set the version